### PR TITLE
added api folder

### PR DESCRIPTION
**Pull request by qjrincoywork**

### CODE REVIEW

File: config/routes.php [hunk 1]
```diff
@@ -46,6 +46,11 @@
 Router::defaultRouteClass(DashedRoute::class);
 
 Router::scope('/', function (RouteBuilder $routes) {
+    Router::prefix('/api', function (RouteBuilder $routes) {
```
Comment (Jodan21) [inline config/routes.php:49]: Move this block of code outside scope('/'....

File: src/Controller/Api/AppController.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    public function beforeRender(Event $event) {
+        $auth = $this->request->getSession()->read('Auth.User');
+        $myId = $this->request->getSession()->read('Auth.User.id');
+        $systemLogo = "/img/microbloglogo.png";
+        
+        $this->set(compact('auth', 'myId', 'systemLogo'));
+    }
+
+    public function formErrors($data) {
```
Comment (Jodan21) [inline src/Controller/Api/AppController.php:85]: Remove loadComponent
and remove other function except initialize, jsonResponse

**Review by Jodan21 — CHANGES_REQUESTED** [review]

File: config/routes.php (hunk 1 above)
Comment (qjrincoywork) [reply config/routes.php:49]: done po

File: src/Controller/Api/AppController.php (hunk 2 above)
Comment (qjrincoywork) [reply src/Controller/Api/AppController.php:85]: done po

**Pull request merged**